public/js/addEvent.js

```diff
@@ -47,6 +46,7 @@ let getCompletedFormData = () => {
     let data = {
         name: eventForm.elements['event-name'].value,
         category: eventForm.elements['event-category'].value,
+        availableToHP: eventForm.elements['available-to-HP'].checked ? 1 : 0,
```

Comment:
do you need the ternary? it will contain true or false anyway

Reply:
false wasn't passing in as zerothrough mysql - this made it happy